platform/linux-dpdk/include/odp/api/feature.h

```diff
@@ -0,0 +1 @@
+../../../../linux-generic/include/odp/api/feature.h
```

Comment:
These sort of absolute relative path expressions will cause trouble as we start to reorganize the directory structure. In addition to causing checkpatch false positives, an m4 solution to these sort of path specifications would seem to be more flexible in this regard.